src/lang/isArray.js

```diff
@@ -0,0 +1,8 @@
+const isArray = (value) => {
+  if (Array.isArray(value)) {
```

Comment:
Try and see if you can solve this without Array.isArray() since it's technically the same thing. 
Some ideas that come to mind are checking if `value.length` is undefined or something.